### ISSUE TITLE
BUG v6.6.0 window is no defined in nodejs environment 

### ISSUE THREAD

**Issue by sethomas** [supasate/connected-react-router#376]
Upon a new deploy to heroku, I am getting the below error for my nodejs app utilizing connected-react-router 
```
2019-11-18T03:09:55.179726+00:00 app[web.1]: ReferenceError: window is not defined
2019-11-18T03:09:55.179728+00:00 app[web.1]: at injectQuery (/app/node_modules/connected-react-router/lib/reducer.js:31:40)
2019-11-18T03:09:55.179730+00:00 app[web.1]: at createRouterReducer (/app/node_modules/connected-react-router/lib/reducer.js:64:17)
```

package.json has this:
```
"connected-react-router": "^6.3.2",
```
Resolution I did was just remove the `^` and good to go.

**Comment by sethomas** [comment]
```
node_modules/connected-react-router/lib/reducer.js:31
  var searchQuery = location.search || window.location.search;
                                       ^

ReferenceError: window is not defined
    at injectQuery 
```
this is on `connected-react-router@6.6.0`

**Comment by sethomas** [comment]
this is not a problem with `v6.5.2`

**Comment by fiendfiend** [comment]
experiencing same issue after upgrading

**Comment by zdila** [comment]
Same for me, downgrading to 6.5.2.

**Comment by KorobkaVadim** [comment]
https://github.com/supasate/connected-react-router/pull/377

**Comment by ivan-aksamentov** [comment]
```sh
yarn add supasate/connected-react-router#pull/377/head
```

For those who live on the edge ;)

**Comment by supasate** [comment]
Fixed in v6.6.1.
